src/mpc/mpc_dummy_phase2.cpp

```diff
@@ -13,8 +14,8 @@ namespace
 //     mpc dummy_phase2 [<option>] <linear_combination_file>
 //
 // Options:
-//     -h,--help           This message
-//     --out <file>        Write phase2 to <file> (mpc-phase2.bin)
+//     -h,--help       This message
```

Comment:
`This message`? Why not just say `-h,--help       Displays help message` or something like that?